### DISCUSSION TITLE
CPM normalization on pooled bootstrapped scRNA-seq data

### DISCUSSION THREAD

**Discussion by Jade-Jung** [SysBioChalmers/Human-GEM#705] [Q&A] [answered]
Hello, I have a question about converting scRNA-seq count data into CPM. 
In the document (https://sysbiochalmers.github.io/Human-GEM-guide/gem\_extraction\_sc/), there is an example converting the pooled NKCells scRNA-sea data into CPM, but not about pooled bootstrapped data.

Is it recommendable to convert pooled bootstrapped data into CPM then use it to extract context-specific model? or just use pooled bootstrapped data into ftINIT?
Thank you.

**Comment by haowang-bioinfo** [comment]
@johan-gson maybe you can help

**Reply by johan-gson** [reply, marked as answer]
Hi, yes, convert to CPM

**Reply by Jade-Jung** [reply]
Thank you so much!
